pages/challenge.tsx

```diff
@@ -0,0 +1,66 @@
+import Head from "next/head"
+import { Table } from "@nextui-org/react"
+
+export default function Web() {
+  return (
+    <>
+      <Head>
+        <meta property="og:url" content="https://next-enterprise.vercel.app/" />
+        <meta
+          property="og:image"
+          content="https://raw.githubusercontent.com/Blazity/next-enterprise/main/project-logo.png"
+        />
+        <meta property="og:image:width" content="1200" />
+        <meta property="og:image:height" content="630" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <title>Oles eleven</title>
+      </Head>
+      <section className="bg-white dark:bg-gray-900">
+        <div className="mx-auto grid max-w-screen-xl px-4 py-8 text-center lg:py-16">
+          <div className="mx-auto place-self-center">
+            <h1 className="mb-4 max-w-2xl text-4xl font-extrabold leading-none tracking-tight dark:text-white md:text-5xl xl:text-6xl">
+              11 Miles challenge
+            </h1>
+            <p className="mb-6 max-w-2xl font-light text-gray-500 dark:text-gray-400 md:text-lg lg:mb-8 lg:text-xl">
+              This challenge is to complete 11 miles (110 kilometers) in under 24 hours by walking and running. It must
+              include atleast one hour of running. it can not include any othe form of transportation.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section>
+        <Table
+          aria-label="Example table with static content"
+          css={{
+            height: "auto",
+            minWidth: "100%",
+          }}
+        >
+          <Table.Header>
+            <Table.Column>Name</Table.Column>
+            <Table.Column>Date</Table.Column>
+            <Table.Column>Number of kilometers</Table.Column>
+            <Table.Column>Kilometers run</Table.Column>
+            <Table.Column>Completed</Table.Column>
+          </Table.Header>
+          <Table.Body>
+            <Table.Row key="1">
+              <Table.Cell>Ole</Table.Cell>
+              <Table.Cell>13.07.2023</Table.Cell>
+              <Table.Cell>70km</Table.Cell>
+              <Table.Cell>10km</Table.Cell>
+              <Table.Cell>No</Table.Cell>
+            </Table.Row>
+            <Table.Row key="2">
+              <Table.Cell>Benjamin</Table.Cell>
+              <Table.Cell>13.07.2023</Table.Cell>
+              <Table.Cell>70km</Table.Cell>
+              <Table.Cell>10km</Table.Cell>
+              <Table.Cell>No</Table.Cell>
+            </Table.Row>
+          </Table.Body>
+        </Table>
```

Comment:
Should refactor.
Suggest making the Leaderboard component in separate file
Suggest making Table header component in separate file
Suggest making Table row component in separate file
Suggest pulling out data into a json-file that is iterated over to generate rows

Reply:
done